data/yaml/venues/parlaclariniv.yaml

```diff
@@ -0,0 +1,3 @@
+acronym: ParlaCLARINIV
+name: ParlaCLARIN IV Workshop on Creating, Analysing, and Increasing Accessibility
+  of Parliamentary Corpora
```

Comment:
Only "Workshop on..."

Reply:
Is "IV" here meaning "fourth"? If so it shouldn't be part of the venue ID

Reply:
Yeah, the venue id should different. It seems [ParlaCLARIN already exists](https://aclanthology.org/venues/parlaclarin/).

Reply:
Done.